catalogue/webapp/pages/item.js

```diff
@@ -123,77 +293,67 @@ const ItemPage = ({
       imageAltText={''}
       hideNewsletterPromo={true}
     >
-      <Layout12>
-        <Paginator
-          currentPage={canvasIndex + 1}
-          pageSize={1}
-          totalResults={canvases.length}
-          linkKey={'canvas'}
-          link={itemUrl({
-            workId,
-            query,
-            page: pageIndex + 1,
-            canvas: canvasIndex + 1,
-            workType,
-            itemsLocationsLocationType,
-            sierraId,
-          })}
-          render={Pagination}
-        />
-      </Layout12>
-      <Layout12>
-        <img
-          className={classNames({
-            'block h-center': true,
-            [spacing({ s: 2 }, { margin: ['bottom'] })]: true,
-          })}
-          style={{
-            maxHeight: '80vh',
-            width: 'auto',
-          }}
-          width={largestSize.width}
-          height={largestSize.height}
-          src={urlTemplate({
-            size: `${largestSize.width},${largestSize.height}`,
-          })}
-        />
-        <Paginator
-          currentPage={pageIndex + 1}
-          pageSize={5}
-          totalResults={canvases.length}
-          linkKey={'page'}
-          link={itemUrl({
-            workId,
-            query,
-            page: pageIndex + 1,
-            canvas: canvasIndex + 1,
-            workType,
-            itemsLocationsLocationType,
-            sierraId,
-          })}
-          render={Pagination}
-        />
-        <div className={classNames({ flex: true })}>
+      <ItemContainer>
```

Comment:
Can't say this name is very descriptive?

Reply:
Probably – do @wellcometrust/experience-devs have any suggestions?

Reply:
We could start splitting this out into what it is, i.e. `IIIFViewer`? 
I don't suppose that is going to change in terms of needing something to view IIIF things.